scripts/lib/mkcloud-common.sh

```diff
@@ -181,14 +182,17 @@ function nodes
     local query=${1:number}
     local type=${2:all}
     local start_id=1
-    local end_id=$(($nodenumber + $nodenumberlonelynode))
+    local end_id=$(($nodenumber + $nodenumberlonelynode + $nodenumberironicnode))
     case $type in
         normal|pxe)
             end_id=$nodenumber
         ;;
         lonely|preinstalled)
             start_id=$(($nodenumber + 1))
         ;;
+        ironic)
+            start_id=$(($nodenumber + $nodenumberlonelynode + 1))
```

Comment:
Oh, new assumptions about node numbers and node types ...
Thats a topic that I am currently working on - but ok, that should work for now.